sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -7,8 +7,9 @@ For users new to the JavaScript SDK for Event Hubs, please see the [readme file
 
 ## General changes
 
-EventHubs v5 is a redesign that provides a better out-of-the-box experience 
-for developers across several areas:
+Version 5 of the `@azure/event-hubs` package is a result of our efforts to create a client library that is user friendly and idiomatic to the JavaScript ecosystem. Apart from redesigns resulting from following the new [Azure SDK Design Guidelines for Typescript](https://azure.github.io/azure-sdk/typescript_introduction.html#design-principles), the latest version provides the below improvements:
+for providing a better out-of-the-box experience for JavaScript developers 
+across several areas:
```

Comment:
lol... This was a test @richardpark-msft 

The above 2 lines need to be deleted now that my other feedback is "incorporated"

Reply:
I'm actually not sure what you mean. The ##general changes should be deleted? It groups together the remaining line items. Did you want this in the "introduction" paragraph instead?

Reply:
I've pushed another update - looking at the change in an editor made me see some other small changes.